.changeset/cold-bananas-poke.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ultraviolet/ui": patch
+---
+
+`<DateInput />`: new prop `input` to show calendar as a card, with no input
```

Comment:
You also need to patch DateField from forms. currently the prop `input` is not spread so we cannot use this new feature in forms.